.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8

```

Comment:
I am a little worried that this would affect all kind of files (incl. binary files).

Reply:
Unless there are binary tools that read .editorconfig files I don't think it does